sopel/modules/unicode_info.py

```diff
@@ -5,22 +5,48 @@
 # Licensed under the Eiffel Forum License 2.
 from __future__ import unicode_literals, absolute_import, print_function, division
 
-import unicodedata
 import sys
-from sopel.module import commands, example, NOLIMIT
+import unicodedata
+
+from sopel import module
 
 if sys.version_info.major >= 3:
+    # Note on unicode and str (required for py2 compatibility)
+    # the `hex` function returns an `str`, both in py2 and py3
```

Comment:
```suggestion
    # the `hex` function returns a `str`, both in py2 and py3
```

Reply:
I swear I tried really hard to pay attention to these details. :stuck_out_tongue: